src/buffer/out/TextColor.h

```diff
@@ -39,10 +39,10 @@ Revision History:
 
 enum class ColorType : BYTE
 {
-    IsIndex256 = 0x0,
-    IsIndex16 = 0x1,
-    IsDefault = 0x2,
-    IsRgb = 0x3
+    IsDefault,
+    IsIndex16,
+    IsIndex256,
+    IsRgb

```

Comment:
The more consistent ordering from "less available colors" to "more" allows for better compiler optimizations, as mentioned above, and also IMO make more sense subjectively speaking.